src/VisualStudio/CSharp/Test/CSharpVisualStudioTest.csproj

```diff
@@ -11,6 +11,7 @@
     <TargetFramework>net461</TargetFramework>
     <RuntimeIdentifier>$(RoslynDesktopRuntimeIdentifier)</RuntimeIdentifier>
     <RoslynProjectType>UnitTest</RoslynProjectType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
that's scary :)

Reply:
Where's your sense of adventure? The unsafer, the better!

Reply:
Our C# library already had unsafe for some fun marshalling hacks we have to do. One thing I was repeatedly being bit by while refactoring this is a lot of unit tests were calling internal members of AbstractProject directly instead of through the interfaces the project system code actually calls through. Fixing this meant our unit tests now have to do some unsafe code to call into the existing unsafe code. :smile: